framework/doc/content/framework/contributing.md

```diff
@@ -13,6 +13,10 @@ engaging in the community will certainly be beneficial.
 
 !alert-end!
 
+!alert note title=Authorship in MOOSE-development papers
+MOOSE being a collaborative effort, the authorship of MOOSE is shared within the community. Please see the [authorship_guidelines.md](framework_development/authorship_guidelines.md optional=True)
```

Comment:
if it still doesnt work I might just hard code. This would actually make it work for all downstream apps ;)

Reply:
works now! thanks for checking @hugary1995